### DISCUSSION TITLE
How to apply new colorscheme theme?

### DISCUSSION THREAD

**Discussion by notj0n** [ecosse3/nvim#159] [Q&A] [answered]
Hello everyone, 
Sorry for the silly question, I'm a beginner. I love the easy to use Ecovim setup but I have my favourite theme (tomorrow-night) and I would like to change it. I know it might an easy task but I don't how to proceed. 
Thanks all

**Comment by alfredosdpiii** [comment, marked as answer]
in `plugins.lua`
Don't forget to comment out the original colorscheme and replace with something like this
 
``` lua
{
    'author/colorschemeName',
    lazy=false,
    priority=1000,
    config = function()
      vim.cmd([[colorscheme colorschemeName]])
    end,
  },
```

**Comment by notj0n** [comment]
I have edit the `plugins.lua` with the following snippet and comment the previous one : 
```
{
    'chriskempson/tomorrow-theme',
    lazy=false,
    priority=1000,
    config = function()
      vim.cmd([[colorscheme tomorrow-theme]])
    end,
},
```

Do I have as well to change the `EcoVim.lua` and `lazy.lua` ? Because it doesn't seem to work

**Comment by ilyaliao** [comment]
Change the colorscheme to `tomorrow-theme`

https://github.com/ecosse3/nvim/blob/e8418eb65af4471891ea9a5b74a94205804c49aa/lua/config/EcoVim.lua#L11-L67
